go/cmd/dolt/cli/arg_parser_helpers.go

```diff
@@ -321,6 +321,7 @@ func CreateProfileArgParser() *argparser.ArgParser {
 	ap.SupportsString("privilege-file", "", "privilege-file", "DePath to a file to load and store users and grants.")
 	ap.SupportsString("branch-control-file", "", "branch-control-file", "Path to a file to load and store branch control permissions.")
 	ap.SupportsString("use-db", "", "use-db", "The name of the database to use when executing SQL queries.")
+	ap.SupportsString("profile", "", "profile", "The name of the profile to use when executing SQL queries. Run `dolt profile --help` for more information.")
```

Comment:
This options should only be available as a global argument. Wouldn't make sense to do the following:

`dolt profile --host foo --profile bar newprofile`

Add a boolean flag to the method to toggle this.

Reply:
Also, make the profile argument be the first one.